test/api/buffer/receivers/delete/007-list-count-1/request.json

```diff
@@ -1,8 +1,7 @@
 {
   "path": "/v1/receivers",
   "method": "GET",
-  "body": {
-    "name": "Github Pull Requests",
-    "exports": []
+  "headers": {
```

Comment:
Asi typo

Reply:
jj, to som mazal z viacerych, a tu to zostalo